website/docs/create-new-theme.md

```diff
@@ -0,0 +1,47 @@
+---
+id: create-new-theme
+title: Create new WordPress theme
+sidebar_label: Create New Theme
+---
+
+## Requirements
+
+1. [Node.js](https://nodejs.org/en/)
+2. [Composer](https://getcomposer.org/)
+3. [WP-CLI](https://wp-cli.org/)
+4. [Git](https://git-scm.com/)
+
+Please make sure that you have all this packages installed and
```

Comment:
There is a typo. Replace `this` with `these`
Please make sure that you have all these packages installed and